config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Dependency.java

```diff
@@ -6,15 +6,29 @@
 
 package com.ca.apim.gateway.cagatewayconfig.beans;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import java.util.Objects;
 
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY;
+
+@JsonInclude(NON_EMPTY)
 public class Dependency {
     private final String id;
+    @JsonIgnore
     private final Class<? extends GatewayEntity> type;
+    private final String name;
+    private final String entityType;
```

Comment:
Can we rename **entityType** to **type** and existing **type** to **typeClass**?

Reply:
we will change entityType but type is existing field, we will handle refactoring in separate PR

Reply:
done